cmake-modules/AzureGlobalCompileOptions.cmake

```
@@ -14,6 +14,8 @@ if(MSVC)
 
   #https://stackoverflow.com/questions/37527946/warning-unreferenced-inline-function-has-been-removed
   add_compile_options(/permissive- /W4 ${WARNINGS_AS_ERRORS_FLAG} /wd5031 /wd4668 /wd4820 /wd4255 /wd4710)
+  #https://learn.microsoft.com/en-us/cpp/build/reference/zc-cplusplus?view=msvc-170
```

```suggestion
  #https://learn.microsoft.com/cpp/build/reference/zc-cplusplus?view=msvc-170
```

Or, maybe drop the comment all together - it is self-evident. If someone has questions, they'll search for "/Zc:__cplusplus", or for the meaning of that MSVC compiler option, and will find exactly the same article.

---

After all, we don't put links to `/wd5031`, `/permissive-` and others.

---

True, though it doesn't hurt. Given it was surprising to me, and we missed it, maybe we keep it and save future Ahson from searching :p